### PR TITLE
Add network related tags

### PR DESCRIPTION
**Pull request by liangxia**
/cc @zhaozhanqi @jhou1 @dis016 @pruan-rht @JianLi-RH

### CODE REVIEW

**Comment by jhou1** [comment]
looks good to me.
/assign @zhaozhanqi

File: features/networking/sgw-lgw-migration.feature [hunk 1]
```diff
@@ -186,7 +186,7 @@ Feature: SGW<->LGW migration related scenarios
   @4.11 @4.10
   @admin
   @destructive
-  @network-ovnkubernetes
+  @network-ovnkubernetes @network-openshiftsdn
```
Comment (zhaozhanqi) [inline features/networking/sgw-lgw-migration.feature:189]: could you remove `@network-openshiftsdn` tag,  I know it's not your tools issue. it's case parameter have `ALL`. I just update this in Polartion

Comment (liangxia) [reply features/networking/sgw-lgw-migration.feature]: @zhaozhanqi Updated.

Comment (zhaozhanqi) [reply features/networking/sgw-lgw-migration.feature]: Thanks

**Pull request merged**